app/controllers/application_controller.rb

```diff
@@ -8,6 +8,7 @@ class ApplicationController < ActionController::Base
   before_action :expose_config
 
   # Gladly adapted from http://sourcey.com/rails-4-omniauth-using-devise-with-twitter-facebook-and-linkedin/
+  # wird nur im partips controller verwendet
```

Comment:
This is meant to be used in all controllers. We won't move it to partips, just so we can move it back once the use case arises.